app/views/pages/privacy_policy.html.erb

```diff
@@ -1,49 +1,49 @@
 <h1 class="heading">Privacy Policy for <strong>Workshop.codes</strong></h1>
 
-<p>This page details what data we collect, what we do with that data, how long we keep that data, and how you can request to delete this data.</p>
+<p>This page details the data we collect, what we do with collected data, how long we keep that data, and how you can request the deletion of said data.</p>
 
-<p>To keep it short; We keep data collecting to a minimum and we keep it non-personal. Email addresses are always optional and are encrypted where used.</p>
+<p>Long story short, we keep data collection to a minimum and keep it as impersonal as possible. The only personally identifiable information we collect is email addresses. Providing an email address is always optional, and the email address is encrypted before being stored.</p>
 
 
 
-<h4 class="mt-1/1">Table of contents</h4>
+<h2 class="mt-1/1">Table of Contents</h2>
 <ul>
-  <li><%= link_to "Data we collect and how we use it", privacy_policy_path(anchor: "data-we-collect") %></li>
+  <li><%= link_to "Data We Collect and How We Use It", privacy_policy_path(anchor: "data-we-collect") %></li>
   <li><%= link_to "Log Files", privacy_policy_path(anchor: "log-files") %></li>
   <li><%= link_to "Cookies", privacy_policy_path(anchor: "cookies") %></li>
   <li><%= link_to "Third Party Privacy Policies", privacy_policy_path(anchor: "third-party-privacy-policies") %></li>
-  <li><%= link_to "Removal of data", privacy_policy_path(anchor: "removal-of-data") %></li>
+  <li><%= link_to "Removal of Data", privacy_policy_path(anchor: "removal-of-data") %></li>
 </ul>
 
 <hr>
 
 
 
-<h2 class="mt-1/1" id="data-we-collect">Data we collect and how we use it</h2>
+<h2 class="mt-1/1" id="data-we-collect">Data We Collect and How We Use It</h2>
 
-<p>When you use the website you are given a token to identify you. This token is completely annonymous and can not be used to identify you personally and expires after 8 hours of no activity. This token is saved in our database and is kept for a maximum of 1 month. An annoymized IP address is saved along with this token. Instead of saving your full IP address the last digits are removed. For example, if your IP address is 192.169.123.45 your IP address is saved as 192.169.123.0.<br></p>
+<p>When you visit Workshop.codes you are given a token (in the form of a browser cookie) to identify you (hereafter referred to as the visitor token). The visitor token is completely anonymous, cannot be used to identify you personally, and expires after 8 hours of no activity. Visitor tokens are saved in our database for a maximum of 1 month. An anonymized IP address is saved along with this token. We anonymize IP addresses by removing the last octet (set of digits after the last period) in the address. For example, if your IP address was 192.169.123.45, your IP address would be saved as 192.169.123.0.<br></p>
```

Comment:
How does this work for ipv6?

Reply:
We use [Ahoy](https://github.com/ankane/ahoy);

> Ahoy can mask IPs with the same approach Google Analytics uses for IP anonymization. This means:
>      For IPv4, the last octet is set to 0 (8.8.4.4 becomes 8.8.4.0)
>      For IPv6, the last 80 bits are set to zeros (2001:4860:4860:0:0:0:0:8844 becomes 2001:4860:4860::)

Reply:
I could probably just link to Ahoy for that for future-proofing I suppose